src/argilla/client/feedback/schemas/questions.py

```diff
@@ -55,7 +55,7 @@ class QuestionSchema(BaseModel):
 
     class Config:
         validate_assignment = True
-        extra = Extra.forbid
+        extra = Extra.ignore
```

Comment:
I would discourage the use of `extra = Extra.ignore` and keep it `Extra.forbid` this way we ensure more robustness, and, is it actually needed?

Reply:
We need to add supporting datasets used in the quickstart image. Otherwise, the process will fail

Reply:
WE should keep the `Extra.ignore` at least for remote schemas. Otherwise, backward API compatibilities won't be applied properly. But this is better to discuss here https://github.com/argilla-io/argilla/pull/3829

Reply:
Those will be applied properly now that we have the `from_api` methods, as we're just keeping what's needed, that's why we included that, to have a better control on what's loaded from the API